src/handbook/peopleops/code-of-conduct.md

```diff
@@ -0,0 +1,121 @@
+   no excuse for poor behavior and poor manners. We might all experience some
+   frustration now and then, but we cannot allow that frustration to turn into a
+   personal attack. It’s important to remember that a company where people feel
+   uncomfortable or threatened is neither productive nor pleasant. FlowForge staff
+   should always be respectful when dealing with other personnel as well as with
+   people outside of FlowForge employment.
+
+### Acceptable and Expected Behavior
+
+The following behaviors are expected and requested of all FlowForge staff:
+
+ - Participate in an authentic and active way. In doing so, you contribute to the
+   health and longevity of FlowForge.
+ - Exercise consideration and respect in your speech and actions at all times.
+ - Attempt collaboration before conflict.
+ - Refrain from demeaning, discriminatory, or harassing behavior and speech.
+ - Be mindful of your surroundings and of your fellow participants. Alert FlowForge
+   leaders if you notice a dangerous situation, someone in distress, or violations
+   of this Code of Conduct, even if they seem inconsequential.
+
+### Unacceptable Behavior
+
+The following behaviors are considered harassment and are unacceptable within our community:
+
+ - Violence, threats of violence or violent language directed against another person.
+ - Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
+ - Posting or displaying sexually explicit or violent material.
+ - Posting or threatening to post other people’s personally identifying information ("doxing").
+ - Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+ - Inappropriate photography or recording.
+ - Inappropriate physical contact. You should have someone’s consent before touching them in any manner.
+ - Unwelcome sexual attention. This includes sexualized comments or jokes; inappropriate touching, groping, and unwelcome sexual advances.
+ - Deliberate intimidation, stalking or following (online or in person).
+ - Advocating for, or encouraging, any of the above behavior.
+ - Repeated harassment of others. In general, if someone asks you to stop, then stop.
+ - Other conduct which could reasonably be considered inappropriate in a professional setting.
+
+### Consequences of Unacceptable Behavior
+
+Unacceptable behavior from any FlowForge staff, including those with
+decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a staff member engages in unacceptable behavior, FlowForge leadership may take
+any action deemed appropriate, up to and including suspension or termination.
+
+### Reporting Violations
+
+If you are subject to or witness unacceptable behavior, or have any other concerns,
+please notify an appropriate member of FlowForge leadership as soon as possible.
```

Comment:
I would make this explicit. The Peopleops manager, CTO, and/or CEO. Preferably at least 2 of them.

This allows everyone to report everyone else to keep C.*O's under the CoC too.

Reply:
Done